### PR TITLE
add unit tests  for profile sections

### PR DESCRIPTION
**Pull request by Tapot**
- add tests for all profile sections
- types fix for Profile and ProfileSection base classes
- remove useless default column mapping creation from `Profile.calculate`
- add some skipped tests for incorrect or missed data for profile sections
- add tests for column mapping
- fix some sections values in generated json

### CODE REVIEW

File: tests/model_profile/sections/test_prob_classification_performance_profile_section.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    assert isinstance(confusion_matrix['values'], list)
+
+
+def test_no_calculation_results() -> None:
+    check_section_without_calculation_results(
+        ProbClassificationPerformanceProfileSection, 'probabilistic_classification_performance'
+    )
+
+
+# @pytest.mark.skip('TODO: fix after ProbClassificationPerformanceAnalyzer fix')
```
Comment (Liraim) [inline tests/model_profile/sections/test_prob_classification_performance_profile_section.py:44]: Remove unused comment?

File: tests/model_profile/sections/test_classification_performance_profile_section.py [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+        reference_data: pandas.DataFrame, current_data: pandas.DataFrame
+) -> None:
+    section_result = calculate_section_results(ClassificationPerformanceProfileSection, reference_data, current_data)
+    check_profile_section_result_common_part(section_result, 'classification_performance')
+    result_data = section_result['data']
+    assert 'metrics' in result_data
+    assert result_data['metrics'] == {}
+
+
+@pytest.mark.skip('TODO: fix errors for analyzers and sections if reference_data is missed')
```
Comment (Liraim) [inline tests/model_profile/sections/test_classification_performance_profile_section.py:91]: Are there some problems to fix this so the test won't be skipped?

**Review by Liraim — CHANGES_REQUESTED** [review]

File: tests/model_profile/sections/test_prob_classification_performance_profile_section.py (hunk 1 above)
Comment (Tapot) [reply tests/model_profile/sections/test_prob_classification_performance_profile_section.py:44]: Oh, yes, I've missed it

File: tests/model_profile/sections/test_classification_performance_profile_section.py (hunk 2 above)
Comment (Tapot) [reply tests/model_profile/sections/test_classification_performance_profile_section.py:91]: I think that the good way to fix it - preprocessing, but I'll try to fix it in the PR

**Review by Liraim — APPROVED** [review]
Looks good

**Pull request merged**